adafruit_vl53l0x.py

```diff
@@ -564,6 +587,60 @@ def range(self):
         self._write_u8(_SYSTEM_INTERRUPT_CLEAR, 0x01)
         return range_mm
 
+    @property
+    def continuous_mode(self):
+        """Is the sensor currently in continuous mode?"""
+        return self._continuous_mode
+
+    @continuous_mode.setter
+    def continuous_mode(self, enabled):
+        if enabled:
+            self.start_continuous()
+        else:
+            self.stop_continuous()
```

Comment:
I wonder if we should make this read-only to force use of start_ and stop_. That way there is only one way to do it.

Furthermore, you could make start_continuous return itself and implement`__enter__` and `__exit__` so it is a context manager. That would achieve something similar to `atexit` (which we don't have in CircuitPython.)

Reply:
hmm interesting... but if we have N VL53L0X sensors, then we will have N `with` statement in the program code though 😨

Reply:
The use of `with` could be optional just in case you want to auto-stop.

Reply:
hmm actually I changed my program from calling `stopContinuous`, to set all of the XSHUT pins to LOW instead...